pkg/render/intrusion_detection.go

```diff
@@ -177,14 +156,23 @@ func (c *intrusionDetectionComponent) Objects() ([]client.Object, []client.Objec
 		pss = PSSPrivileged
 	}
 
-	objs := []client.Object{
-		CreateNamespace(IntrusionDetectionNamespace, c.cfg.Installation.KubernetesProvider, PodSecurityStandard(pss)),
-		c.intrusionDetectionControllerAllowTigeraPolicy(),
-		networkpolicy.AllowTigeraDefaultDeny(IntrusionDetectionNamespace),
+	objs := []client.Object{}
+	if !c.cfg.Tenant.MultiTenant() {
+		// In multi-tenant environments, the namespace is pre-created. So, only create it if we're not in a multi-tenant environment.
+		objs = append(objs, CreateNamespace(c.cfg.Namespace, c.cfg.Installation.KubernetesProvider, PodSecurityStandard(pss)))
+
+		// ES Secrets are only used for the Kibana dashboard installer Job, which is not run in multi-tenant mode.
+		objs = append(objs, secret.ToRuntimeObjects(secret.CopyToNamespace(c.cfg.Namespace, c.cfg.ESSecrets...)...)...)
+
+		// GlobalAlertTemplates are not used in multi-tenant management clusters.
+		objs = append(objs, c.globalAlertTemplates()...)
```

Comment:
Do we also need to disable this feature from the UI for multi-tenancy ? :laughing:

Reply:
Nope, don't think so - this is still supported in managed clusters, just not on the management cluster itself.